.changeset/nice-drinks-do.md

```diff
@@ -0,0 +1,9 @@
+---
+"@hashicorp/design-system-codemods": minor
+"@hashicorp/design-system-components": minor
+---
+
+`Dropdown::ListItem::Interactive`
```

Comment:
Question: if you're including a code mod, should you include that in the changeset notes?